docs/usage/other_features/customizing_homepage.md

```diff
@@ -86,4 +86,16 @@ After running the paragraph, you will see output similar to this one:
 
 That's it! Voila! You have your note list.
 
+<br />
+## Show recent notes in your custom homepage
+To display the recent notes on your custom homepage 
+add the following code to a paragraph in your Apache Zeppelin note and run it.
+
+```javascript
+%spark
 
+println(
+"""%angular
+  <div ng-include="'app/home/recent-notes.html'"></div>
```

Comment:
I know, but it is a little weird for me that user need to run a paragraph via spark interpreter to get this feature. Sorry, personally I don't think it worthwhile to add this kind of feature at the cost of adding more complexity to zeppelin codebase. I will wait for others to comment.